CMake/RealmCore.cmake

```diff
@@ -182,7 +194,7 @@ function(build_realm_sync sync_directory)
         BUILD_IN_SOURCE 1
         BUILD_ALWAYS 1
         CONFIGURE_COMMAND ""
-        BUILD_COMMAND make -C src/realm librealm-sync.a librealm-sync-dbg.a librealm-server.a librealm-server-dbg.a ${MAKE_FLAGS}
+        BUILD_COMMAND CPATH=${REALM_CORE_INCLUDE_DIR} make -C src/realm librealm-sync.a librealm-sync-dbg.a librealm-server.a librealm-server-dbg.a ${MAKE_FLAGS}
```

Comment:
What does this do? AFAIK CPATH is something that CMake looks at. Nothing in core's build system appears to use it.